src/data/sample.json

```diff
@@ -63,95 +84,334 @@
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "12",
+      "name": "Database Access Layer Implementation",
+      "description": "Database Access Layer Implementation",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "13",
+      "name": "CI/CD Pipeline Implementation",
+      "description": "CI/CD Pipeline Implementation",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "14",
+      "name": "Cloud Infrastructure & Services Provisioning",
+      "description": "Cloud Infrastructure & Services Provisioning",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "15",
+      "name": "Backup & Recovery Procedures",
+      "description": "Backup & Recovery Procedures",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "16",
+      "name": "Security Procedures",
+      "description": "Security Procedures",
+      "createdAt": "2023-06-15T08:45:00Z",
+      "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "17",
+      "name": "Product Features Management & Roadmapping",
+      "description": "Product Features Management & Roadmapping",
+      "createdAt": "2023-06-15T08:45:00Z",
+      "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "18",
+      "name": "Product Testing & Feedback Management",
+      "description": "Product Testing & Feedback Management",
+      "createdAt": "2023-06-15T08:45:00Z",
+      "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "19",
+      "name": "Machine Learning",
+      "description": "Machine Learning",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    },
+    
+    {
+      "id": "20",
+      "name": "Deep Learning",
+      "description": "Deep Learning",
+     "createdAt": "2023-06-15T08:45:00Z",
+    "updatedAt": "2023-06-18T10:20:00Z"
+    }
+  ],
+  
+    "interestGroups": [
+      {
+        "id": "1",
+        "name": "Machine Learning & AI",
+        "description": "Interest group focused on Machine Learning and Artificial Intelligence technologies.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      },
+      {
+        "id": "2",
+        "name": "Infrastructure, DevOps & Cloud Engineering",
+        "description": "Interest group covering Infrastructure, DevOps practices, and Cloud Engineering solutions.",
+        "dateCreated": "2024-04-11T12:00:00Z",
+        "updatedAt": "2024-04-11T12:00:00Z"
+      },
+      {
+        "id": "3",
+        "name": "Data Science & Data Engineering",
+        "description": "Interest group dedicated to Data Science and Data Engineering disciplines.",
+        "dateCreated": "2024-04-11T12:00:00Z",
```

Comment:
Looking good! 
Please can you change all instances of `dateCreated` to `createdAt` in all entities.